.github/workflows/workflow-validate-repo-markdown.yml

```diff
@@ -1,70 +0,0 @@
-name: Validate Repo Markdown
```

Comment:
Question, non-blocking: ditto on link checking and team perspective.

Reply:
Restored this since it was needed.